github-release.sh

```diff
@@ -103,7 +103,7 @@ if [ "$PUSH_CHANGES" == 'true' ] && [ -z "$GITHUB_API_TOKEN" ]; then
 	exit 1
 fi
 
-docsUrl="https://docs.jboss.org/hibernate/orm/${RELEASE_VERSION_FAMILY}"
+docsUrl="https://docs.jboss.org/hibernate/${PROJECT}/${RELEASE_VERSION_FAMILY}"
```

Comment:
Hibernate Reactive docs are not hosted on jboss.org... I'd like that, but it's just not how it's done currently.

Reply:
I'm going to revert it.